bundles/org.openhab.binding.irobot/src/main/java/org/openhab/binding/irobot/internal/handler/RoombaHandler.java

```diff
@@ -189,9 +189,38 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                         String[] params = cmds[1].split(";");
 
                         String mapId = params[0];
-                        String[] regionIds = params[1].split(",");
-
-                        MQTTProtocol.Request request = new MQTTProtocol.CleanRoomsRequest("start", mapId, regionIds);
+                        String user_mapv_id;
```

Comment:
Naming convention: userMapvId.
Do not know if it is normal that you have sometimes userPmapvId, sometimes UserMapvId.